<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pb_credit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Обработать Кредит</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pb_free_credit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Отпустить Кредит</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pb_kassa">
      <property name="text">
       <string>Обработать Касса</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pb_free_kassa">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Отпустить кассу</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pb_card">
      <property name="text">
       <string>Обработать Карта</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pb_free_card">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Отпустить Карту</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pb_payment">
      <property name="text">
       <string>Обработать Платежи</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pb_free_payment">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Отпустить Платеж</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
